### PR TITLE
Add URLs to package.json

### PR DESCRIPTION
**Pull request by hagenburger**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,8 +9,12 @@
   },
   "repository": {
     "type": "git",
-    "repository": "git@github.com:livingstyleguide/pimd.git"
+    "repository": "https://github.com/hagenburger/pimd.git"
```
Comment (tobmaster) [inline package.json]: I think this should be simply this (according to the [docs](https://docs.npmjs.com/files/package.json#repository) )
```
 "repository": "github:hagenburger/pimd"
```

Comment (tobmaster) [reply package.json]: Haven't tested that yet but I assume npmjs.org will resolve it correctly then based on context

**Comment by hagenburger** [comment]
@tobmaster done!

**Comment by tobmaster** [comment]
Merge! 😊

**Pull request merged**